app/src/full/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -111,6 +118,29 @@ class MessagingService : FirebaseMessagingService() {
                     Log.d(TAG, "Sending notification title to TTS")
                     speakNotification(it[TITLE])
                 }
+                it[MESSAGE] in DEVICE_COMMANDS -> {
+                    Log.d(TAG, "Processing device command")
+                    when (it[MESSAGE]) {
+                        COMMAND_DND -> {
+                            if (it[TITLE] in DND_COMMANDS) {
+                                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M)
+                                    handleDeviceCommands(it[MESSAGE], it[TITLE])
+                                else {
+                                    mainScope.launch {
+                                        Log.d(TAG, "Posting notification to device as it does not support DND commands")
+                                        sendNotification(it)
+                                    }
+                                }
+                            } else {
+                                mainScope.launch {
+                                    Log.d(TAG, "Invalid DND command received, posting notification to device")
+                                    sendNotification(it)
+                                }
+                            }
+                        }
+                        else -> Log.d(TAG, "No command received")
```

Comment:
Shouldn't we send the notification to be consistent here as well?

Reply:
It technically shouldn't be possible to hit it due to line 121 because we are specifying all device commands there.  My goal was that each command would be added with a new condition and variable per the list above.  I thought about it but opted not to because we should be specific here, I can add it if you think we should.

Reply:
Ok, I see what you mean.  We can keep it the way it is.